packages/trader/tsconfig.json

```diff
@@ -1,40 +1,9 @@
 {
+    "extends": "../../tsconfig.json",
     "compilerOptions": {
-        "paths": {
-            "_common/*": ["src/_common/*"],
-            "App/*": ["src/App/*"],
-            "Assets/*": ["src/Assets/*"],
-            "Constants/*": ["src/Constants/*"],
-            "Fonts/*": ["src/public/fonts/*"],
-            "Images": ["src/public/images"],
-            "Modules/*": ["src/Modules/*"],
-            "Sass/*": ["src/sass/*"],
-            "Services/*": ["src/Services/*"],
-            "Stores/*": ["src/Stores/*"],
-            "Translations/*": ["src/public/translations/*"],
-            "Utils/*": ["src/Utils/*"]
-        },
-        "target": "es5",
-        "lib": ["dom", "dom.iterable", "esnext"],
         "outDir": "./dist",
         "rootDir": "./src",
-        "baseUrl": "./",
-        "allowJs": true,
-        "skipLibCheck": true,
-        "esModuleInterop": true,
-        "allowSyntheticDefaultImports": true,
-        "strict": true,
-        "forceConsistentCasingInFileNames": true,
-        "noFallthroughCasesInSwitch": true,
-        "module": "esnext",
-        "moduleResolution": "node",
-        "resolveJsonModule": true,
-        "isolatedModules": true,
-        "noEmit": true,
-        "jsx": "react",
-        "experimentalDecorators": true,
-        "skipDefaultLibCheck": true
+        "baseUrl": "src"
     },
-    "include": ["./src/**/*.ts", "./src/**/*.tsx", "globals.d.ts"],
-    "exclude": ["**/*.js", "**/*.jsx"]
+    "include": ["./src", "./src/**/*.ts", "./src/**/*.tsx", "globals.d.ts"]
```

Comment:
It would be better to add global declarations in all of our packages and declare the packages that are not using TS yet to avoid further issues. When they start working on TS they would face errors while importing the packages that are still using JS.